webpack.common.config.js

```diff
@@ -91,6 +92,7 @@ var entry = {
 
 // Collect log text from addons
 var mainLogs = require(staticPath('js/logActionsList.json'));
+var anonymousLogs = require(staticPath('js/_anonymousLogTexts.js'));
 var addonLog;
 
 // Collect addons endpoints. If an addon's static folder has
```

Comment:
Note: regarding [this comment](https://github.com/CenterForOpenScience/osf.io/pull/5825/files#r66667783)

>Consider refactoring to have the logs for each addon [in addon folders]

Implementing this should be a simple modification to the nested `forEach` loops below here (plus putting the logs in the right places).